openai/package.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "@winglibs/openai",
+  "description": "openai library for Wing",
+  "version": "0.0.1",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/winglang/winglibs.git",
+    "directory": "openai"
+  },
+  "author": {
+    "email": "shaib@wing.cloud",
+    "name": "Shai Ber"
+  },
+  "license": "MIT",
+  "dependencies": {
```

Should be changed to `peerDependencies` see #5470
```suggestion
  "peerDependencies": {
```

---

OK, so I'm also also updating the template

---

Actually, it is not in the template, but are you sure that it needs to be peerdependecy? I don't see any other winglib with it

---

It should be, it's just not enforced right now https://github.com/winglang/wing/issues/5470